build.gradle

```diff
@@ -84,6 +84,11 @@ dependencies {
     implementation 'com.github.hypfvieh:dbus-java:3.3.1'
     implementation 'com.apollographql.apollo:apollo-runtime:2.5.14'
     implementation 'com.apollographql.apollo:apollo-http-cache:2.5.14'
+    implementation 'com.apollographql.apollo:apollo-rx3-support:2.5.14'
+
+    // RxJava
+    implementation("io.reactivex.rxjava3:rxjava:3.1.6")
+    implementation("com.gitlab.doomsdayrs:rxswing:a5749ad421") // TODO tag
```

Comment:
You mentioned finding https://github.com/akarnokd/RxJavaSwing in Discord. Was that package a no go?

Reply:
It was on life support. I gave it few patches to clean up warnings. I might remember to upstream them later.